build.gradle.kts

```diff
@@ -336,6 +337,17 @@ tasks.register("printSdkDebugRuntimeClasspath") {
             result += File(androidJarFilePath.joinToString(File.separator))
         }
 
+        val envSdkHome = System.getenv("ANDROID_SDK_ROOT")
```

Comment:
This ensures that in CI (where no `local.properties` file exists) we still include the `android.jar` for type resolution.